cmd/vela-worker/exec.go

```diff
@@ -60,6 +62,17 @@ func (w *Worker) exec(index int, config *library.Worker) error {
 		return err
 	}
 
+	execBuildExecutable, resp, err := execBuildClient.Build.GetBuildExecutable(item.Repo.GetOrg(), item.Repo.GetName(), item.Build.GetNumber())
```

Comment:
Can we add a comment above this call to explain what is going on?

Reply:
Yes!